gemrb/GUIScripts/iwd2/MessageWindow.py

```diff
@@ -109,10 +106,8 @@ def UpdateControlStatus():
 			Button.SetFlags(IE_GUI_BUTTON_NO_IMAGE, OP_SET)
 		else:
 			Button.SetPicture(Portrait, "NOPORTSM")
-	else:
-		GUICommon.GameControl.SetStatus(IE_GUI_CONTROL_FOCUSED)
```

Comment:
There used to be a global called `GameControl` that was a reference to the `GameControl`. In general its not needed because we no longer need to manually focus it and whatnot. I think if you do need it there is an alias for it now.